src/options/options.html

```diff
@@ -172,6 +172,13 @@ <h1 data-i18n="__MSG_titleAddonBehavior__">Addon behavior</h1>
 						</div>
 					</li>
 
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="autoGetClipboardContent" name="autoGetClipboardContent">
+							<label data-i18n="__MSG_optionAutoGetClipboardContent__" for="autoGetClipboardContent">Use clipboard content when opening popup</label>
```

Comment:
Yeah good, now only the `optionAutoGetClipboardContent` also needs localization in the `message.json` file. English is fine for now (just duplicate the string there, too).

See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#providing_localized_strings_in__locales for more information.